components/builder-web/app/shared/statuspage/_statuspage-indicator.component.scss

```diff
@@ -0,0 +1,25 @@
+// The CSS for the colored dot that represents your current status
+$red:#e74c3c;
+$orange:#e67e22;
+$yellow:#f1c40f;
+$green:#2ecc71;
+
+.color-dot {
+  border-radius: 99px;
+  display:inline-block;
+  width:10px;
+  height:10px;
+  margin-right:5px;
```

Comment:
We have a `default-margin` variable you should probably use for this. (Most everything is based on some multiple of 8; I might use either `default-margin` or `default-margin / 2` for example.)